package.json

```diff
@@ -71,27 +71,27 @@
     "devDependencies": {
         "chai": "*",
         "chai-as-promised": "^7.1.1",
-        "chai-http": "^4.2.0",
-        "electron": "^2.0.10",
-        "electron-builder": "^20.28.4",
-        "husky": "^0.14.3",
+        "chai-http": "^4.3.0",
+        "electron": "^9.3.0",
```

Comment:
I suspect this will break the app. Build also fails, have you tested this locally?

Reply:
@japborst yes, it works. I've updated tests and manually tested it. But however I cannot make the build pass due to issues with nodejs version. Maybe could you help with that?